pom.xml

```diff
@@ -44,8 +44,9 @@
         <slf4j.version>1.7.30</slf4j.version>
         <log4j.version>2.13.2</log4j.version>
 
-        <grpc.version>1.28.1</grpc.version>
-        <netty.tcnative.version>2.0.28.Final</netty.tcnative.version>
+        <!-- For compatibility between grpc and tcnative, check https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty -->
+        <grpc.version>1.33.0</grpc.version>
```

Comment:
Just to mention that the table says `1.32.x` is compatible and not `1.33.x`.
But if this is working, I am fine.

Reply:
I guess it means "from 1.32.x to the last release"